immuni_exposure_ingestion/apis/ingestion.py

```diff
@@ -104,6 +105,7 @@
 )
 @validate_token_format
 @cache(no_store=True)
+@track_upload
```

Comment:
I'd maybe to two small changes:
(i) Since the whole monitoring, even helpers, in common comes from the dedicated monitoring folder, I'd keep the same pattern also here in immuni_exposure_ingestion.
(ii) I'd rename the methods, changing `track` into `monitor`, for consistency.

Reply:
So you'd suggest we move the `track_upload` and `track_check_otp` into a `monitoring/helpers` module and rename them to `monitor_upload` and `monitor_check_otp` ?

Reply:
Exactly 👍